src/NServiceBus.Transport.Msmq/MsmqQueueCreator.cs

```diff
@@ -8,13 +8,18 @@ namespace NServiceBus.Transport.Msmq
 
     class MsmqQueueCreator : ICreateQueues
     {
-        public MsmqQueueCreator(bool useTransactionalQueues)
+        public MsmqQueueCreator(bool useTransactionalQueues, bool disableInstaller = false)
```

Comment:
Do we need the default value? 

I think it reads better if we just call the parameter `enabled`? (see below)

Reply:
I used the default value here for the 2nd parameter, otherwise, I'd have to change a whole bunch of existing unit tests to go and add the 2nd parameter.

Reply:
I had to revert my changes. The API is called DisableInstaller and it sets a true. The default value for the 2nd parameter in MsmqQueueCreator cannot also be true. After a "few" failing acceptance tests, I've reverted back, but named the variables better in MsmqQueueCreator. Hope this helps.